test/libsolidity/semanticTests/viaYul/require.sol

```diff
@@ -0,0 +1,28 @@
+contract C {
+	function f(bool a) public pure returns (bool x) {
+		bool b = a;
+		x = b;
+		require(b);
+	}
+	function fail() public pure returns (bool x) {
+		x = true;
+		require(false);
+	}
+	function succeed() public pure returns (bool x) {
+		x = true;
+		require(true);
+	}
+	/* Not properly supported by test system yet
```

Comment:
I think we should still have the test and then auto-update the expectation once we can check for it.

Reply:
Unfortunately that is not possible. There is no expectation that passes.

Reply:
You mean just `FAILURE` does not pass?

Reply:
Exactly. Because the same check also checks that the returned data has length 0